cas-server-core-util/src/main/java/org/jasig/cas/util/http/SimpleHttpClientFactoryBean.java

```diff
@@ -428,6 +429,14 @@ public void setRedirectsEnabled(final boolean redirectsEnabled) {
     }
 
     /**
+     * Destroy.
+     */
+    @PreDestroy
+    public void destroy() {
+        this.executorService.shutdownNow();
```

Comment:
I haven't fully looked at how we are currently initializing `executorService`, but it appears that in https://github.com/Unicon/cas/blob/dangling-threads/cas-server-core-util/src/main/java/org/jasig/cas/util/http/SimpleHttpClientFactoryBean.java#L250 we create a `ThreadPoolExecutor` but don't store it locally where we could shut it down.